setup.py

```diff
@@ -53,6 +53,8 @@
         'google-api-python-client==2.57.0',  # google drive
         'google-auth-httplib2==0.1.0',  # google drive
         'google-auth-oauthlib==0.5.2',  # google drive
+        'google-cloud-billing==1.9.1',
+        'google-cloud-bigquery==3.5.0',
```

Comment:
add comment that its for google cloud cost for both 
        'google-cloud-billing==1.9.1', # google cloud cost 
        'google-cloud-bigquery==3.5.0', # google cloud cost